README.md

```diff
@@ -85,3 +90,16 @@ Discuss features and ask questions on
 ## License
 
 MIT
+
+## Module Long Term Support Policy
+This module adopts the [Module Long Term Support (LTS)](http://github.com/CloudNativeJS/ModuleLTS) policy, with the following End Of Life (EOL) dates:
+
+| Version | Status          | Published | EOL      |
+| ------- | --------------- | --------- | -------- |
+| 5.x     | Active LTS      | Nov 2018  | Dec 2019 |
+| 4.x     | Maintenance LTS | Sep 2017  | Apr 2019 |
```

Comment:
Earlier this year (e.g. in May), if a developer wanted to pick the latest Node.js & loopback-cli versions for their projects, they would have used Node.js 8.x and loopback-cli@4. Thus we need to keep loopback-cli@4 supported as long as Node.js 8.x is supported (Dec 2019).